CGCharter-1727386911.html

```diff
@@ -464,6 +461,14 @@ <h2 id="offboarding-chairs">
       </li>
     </ul>
 
+    <h2 id="committers">
+      Committers
+    </h2>
+
+    <p>
+      A <dfn>Committer</dfn> is authorized to make changes to one or more GitHub repositories that are managed by the group. Participants can earn Committer status through a history of useful contributions. Granting Committer status is made through consensus, as discussed in <a href="#decision">Decision Process</a>. There is no limit to the number of Committers in the group or on a repository. There is no time limit on Committer status. Each repository should have at least one active Committer. Chair(s) can revoke Committer status for any reason. A Committer can give up the role voluntarily.
```

Comment:
```suggestion
      A <dfn>Committer</dfn> is authorized to make changes to one or more GitHub repositories that are managed by the group. Participants can earn Committer status through a history of useful contributions. Granting Committer status is made through consensus, as discussed in <a href="#decision">Decision Process</a>. There is no limit to the number of Committers in the group or on a repository. There is no time limit on Committer status. Each repository for ongoing work items should have at least one active Committer other than the Chairs. Chair(s) can revoke Committer status for any reason. A Committer can give up the role voluntarily.
```

Reply:
i'm not wed to this little suggestion, and it kinda goes without saying since all work items have "champions" in the staging process anyways, but thought it might be a helpful reminder to give write privileges to one or more key participants any time a new repo is made 😄